Cargo.toml

```diff
@@ -30,6 +30,7 @@ trippy-privilege = { version = "0.13.0-dev", path = "crates/trippy-privilege" }
 trippy-dns = { version = "0.13.0-dev", path = "crates/trippy-dns" }
 trippy-packet = { version = "0.13.0-dev", path = "crates/trippy-packet" }
 anyhow = "1.0.91"
+arc-swap = "1.7.1"
```

Comment:
Using `ArcSwap` rather than `RwLock`, but as an uncontended lock I don't image there there is an real world performance hit here. Maybe `RwLock` is better to avoid the dependency?

Reply:
`RwLock` is just a r+w lock I happen to know ;) arc-swap is in Debian so not a problem in that regard.